content/docs/user-guide/plots.md

```diff
@@ -0,0 +1,420 @@
+Plots are a great alternative to `dvc metrics` when working with
+multi-dimensional performance data. They also help you present and compare
+[experiments] effectively.
+
+DVC can work with two types of plots files:
+
+1. Data series files, which can be JSON, YAML, CSV or TSV.
+2. Image files in JPEG, GIF, or PNG format.
+
+DVC generates plots as static HTML webpages you can open with a web browser
+(they can be saved as SVG or PNG image files from there). You can also visualize
+DVC plots from the [VS Code Extension], which includes a special [Plots
+Dashboard] that corresponds to the features in the `dvc plots` commands.
+
+Data-series plots utilize [Vega-Lite](https://vega.github.io/vega-lite/) for
+rendering (declarative JSON grammar for defining graphics). Images are rendered
+using `<img>` tags directly.
+
+[vs code extension]:
+  https://marketplace.visualstudio.com/items?itemName=Iterative.dvc
+[plots dashboard]:
+  https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/plots.md
+[experiments]: /doc/user-guide/experiment-management/experiments-overview
+
+### Supported file formats
+
+Images are included in HTML as-is, without additional processing.
+
+> We recommend to track these source image files with DVC instead of Git, to
+> prevent the repository from bloating.
+
+Structured plots can be read from JSON, YAML 1.2, CSV, or TSV files. DVC expects
+to see an array (or multiple arrays) of objects (usually _float numbers_) in the
+file.
+
+In tabular file formats such as CSV and TSV, each column is an array.
+`dvc plots` subcommands can produce plots for a specified column or a set of
+them. For example, `epoch`, `AUC`, and `loss` are the column names below:
+
+```
+epoch, AUC, loss
+34, 0.91935, 0.0317345
+35, 0.91913, 0.0317829
+36, 0.92256, 0.0304632
+37, 0.92302, 0.0299015
+```
+
+Hierarchical file formats such as JSON and YAML consists of an array of
+consistent objects (sharing a common structure): All objects should contain the
+fields used for the X and Y axis of the plot (see
+[DVC template anchors](/doc/user-guide/plots#custom-templates)); Extra elements
+will be ignored silently.
+
+`dvc plots` subcommands can produce plots for a specified field or a set of
+them, from the array's objects. For example, `val_loss` is one of the field
+names in the `train` array below:
+
+```json
+{
+  "train": [
+    { "val_accuracy": 0.9665, "val_loss": 0.10757 },
+    { "val_accuracy": 0.9764, "val_loss": 0.07324 },
+    { "val_accuracy": 0.877, "val_loss": 0.08136 },
+    { "val_accuracy": 0.874, "val_loss": 0.09026 },
+    { "val_accuracy": 0.8795, "val_loss": 0.0764 },
+    { "val_accuracy": 0.8803, "val_loss": 0.07608 },
+    { "val_accuracy": 0.8987, "val_loss": 0.08455 }
+  ]
+}
+```
+
+## Defining plots
+
+In order to create visualizations, users need to provide the data and
+(optionally) configuration that will help customize the plot. DVC provides two
```

Comment:
... what I'm suggesting to summarize (heavily probably) and move to the dvc.yaml spec file is everything in this section. It's mainly about the schema.

Reply:
> use the guide to show how to setup plots and leave the cmd ref and dvc.yaml pages for lookup purposes

Reply:
> 2... use the guide to show how to setup plots and leave the cmd ref and dvc.yaml pages for lookup purposes

We agree on this @dberenbaum. I just think the [Top-level plots](https://dvc-org-guide-plots-u9q3eipvdr.herokuapp.com/doc/user-guide/plots#top-level-plots) explanation can be shorter and leave most schema details [for the dvc.yaml "guide"](https://dvc-org-guide-plots-u9q3eipvdr.herokuapp.com/doc/user-guide/project-structure/dvcyaml-files#top-level-plot-definitions).

E.g. phrases like "should be defined ... under plots key" or "Configuration ... should be a dictionary", and the whole **Available configuration fields** section (as noted). In short, this section is currently a more complete ref than the actual ref.